### PR TITLE
🍕 package.json fixups

### PR DESCRIPTION
**Pull request by samccone**
:+1: let me know what you think....

Thanks! :cactus: :cake:

### CODE REVIEW

**Comment by abdonrd** [comment]
A description much more in line! +1

**Comment by samccone** [comment]
rebased on top of master, let me know if there is anything else

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Polymer",
   "version": "1.1.3",
-  "description": "Authors interested in learning the core concepts in 0.8 may be interested in our [primer](https://github.com/Polymer/polymer/blob/0.8-preview/PRIMER.md).",
+  "description": "The Polymer library makes it easy to create your own powerful elements. Give your element some markup and properties, and then use it on a site. Polymer provides useful features like templating and data binding to reduce the amount of boilerplate you need to write",
```
Comment (ebidel) [inline package.json]: your own web components

Comment (ebidel) [inline package.json]: features like dynamic templates and data binding...

**Comment by ebidel** [comment]
@azakus can you make sure you're happy with these changes?

**Comment by azakus** [comment]
LGTM

**Comment by samccone** [comment]
all set @ebidel, also I took the description from the www so we should update the copy there also

**Comment by ebidel** [comment]
LGTM x 2

**Comment by samccone** [comment]
:pizza: :pizza: :pizza: :pizza: :pizza: :pizza: :pizza: :pizza: :pizza: :pizza:  all around

**Pull request merged**